app/backend/test/teet/activity_commands_test.clj

```diff
@@ -197,30 +197,12 @@
                             :task/group :task.group/land-purchase}})))
 
 (deftest delete-activity
-  (testing "Activity can be deleted if doesn't have a procurement number"
+  (testing "Activity can be deleted"
     (tu/give-admin-permission tu/mock-user-boss)
     (create-new-activity)
-
-    ;; Let's give the activity a procurement number. In reality this would come from THK.
-    (tu/tx {:db/id (tu/get-data :new-activity-id)
-            :activity/procurement-nr "123 R 456"})
-
-    (is (thrown? Exception
-                 (tu/local-command tu/mock-user-boss
-                                   :activity/delete
-                                   {:db/id (tu/get-data :new-activity-id)})))
-
-    (is (not (:meta/deleted? (du/entity (tu/db)
-                                        (tu/get-data :new-activity-id)))))
```

Comment:
you could add a check in this checks place that makes sure non admins can't delete activities.

Reply:
So, it should be added to AC first

Reply:
> Admins can delete all activities that don't have files (remove the procurement constraint mentioned above)

Is it not in the ac?

Reply:
It is in the AC. As we agreed we don't test auth. matrix?